synapse/lib/boss.py

```diff
@@ -27,7 +27,14 @@ def ps(self):
     def get(self, iden):
         return self.tasks.get(iden)
 
-    async def promote(self, name, user, info=None, taskiden=None):
+    def getRoot(self):
```

Comment:
I called this `getRoot()` because the boss/task term is root but it might make more sense as `getParent()`?